main.py

```diff
@@ -134,4 +134,4 @@ def router(url):
         return jumbotron
 
 if __name__ == "__main__":
-    app.run_server(debug=True, port=8050)
+    app.run_server(debug=True, host='0.0.0.0', port=8050)
```

Comment:
Disable the debug mode when running the app in the container.
Research about FLASK env vars, for instance: FLASK_DEBUG_MODE.